src/AWS.Deploy.Common/Recipes/Validation/IOptionSettingItemValidator.cs

```diff
@@ -10,6 +10,12 @@ namespace AWS.Deploy.Common.Recipes.Validation
     /// </summary>
     public interface IOptionSettingItemValidator
     {
-        ValidationResult Validate(object input);
+        /// <summary>
+        /// Validates an override value for an OptionSetting
```

Comment:
nit: add a ref to OptionSettingItem object. i.e <see cref="OptionSettingItem"/>

Reply:
Done